studio/schemas/documents/compensations.ts

```diff
@@ -1,16 +1,31 @@
 import { defineField, defineType } from "sanity";
 import { titleSlug } from "../schemaTypes/slug";
 import seo from "../objects/seo";
+import offices from "../objects/offices";
 import { title } from "../fields/text";
 import { benefitId } from "./benefit";
 
+/*
+ maximum number of offices without preview string shortening
+ (should be at least 2)
+ given array: ["Trondheim", "Oslo", "Bergen", "Molde"]
+ 2 => Trondheim, and more
+ 3 => Trondheim, Oslo, and more
+ 4 => Trondheim, Oslo, Bergen, and Molde
+*/
+const OFFICES_PREVIEW_CUTOFF = 3;
+
 export const compensationsId = "compensations";
 
 const compensations = defineType({
   name: compensationsId,
   type: "document",
   title: "Compensations",
   fields: [
+    {
+      ...offices,
+      description: "Which offices are these compensations relevant for?",
```

Comment:
I like what you had before. I think we can adjust if we see fit later. I added suggestions on title and descriptions down below 👇